server/controller/ResultController.js

```diff
@@ -166,6 +166,35 @@ router.get(
   }
 );
 
+// @desc Gets the result for the early bird ranking (first N flights of the season)
```

Comment:
Es gibt zwar jetzt die Earlybird Wertung, aber in die Gesamtwertung fließt diese noch garnicht ein, oder?
Und warum ist die airspaces Datei rausgeflogen?

Reply:
Die Punkte fließen aktuell auch nicht in die Gesmtwertung ein.

Die werden zwar da in der Übersicht auf die Punkte des Fluges angerechnet, aber das hat sonst nirgens Konsequenzen. 

Es gibt jetzt zwei Dateien. 
1x airspaces-deu
1x airspaces-lux

Reply:
Bist du sicher? Es gibt doch 20 Punkte für den ersten der Wertung für die Gesamtwertung.